back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/importers/project_importer.rb

```diff
@@ -79,10 +82,40 @@ def import_users(users, user_custom_fields)
           log "ERROR importing user '#{user_row[USER_EMAIL]}': #{e.message}"
         end
         log "Imported #{num_created} new users (#{num_existing} existing users were skipped)"
-        log 'CUSTOM FIELDS NOT IMPLEMENTED YET'
       end
     end
 
+    # NOTE: Pinched a little from IdeaBaseFileParser. Could not think of way to share the code easily.
+    # Currently only works for fields we know we need to import for new west
+    def process_user_custom_field_values(custom_fields, user_row)
+      custom_field_values = {}
+      user_row.each do |field_name, value|
+        field = find_object_by_title(custom_fields, field_name)
+        next unless field && value
+
+        # If the field is a select or multiselect, we need to convert the value to the option keys
+        if SELECT_TYPES.include? field[:input_type]
+          split_values = value.to_s.split('; ')
+          option_keys = []
+          split_values.each do |option_title|
+            option = find_object_by_title(field[:options], option_title)
+            option_keys << option[:key] if option
+          end
+
+          value = option_keys.compact.uniq
+          value = value.first if field[:input_type] == 'select' && value.is_a?(Array)
+        elsif field[:input_type] == 'number'
+          value = value.to_i
+        end
+        custom_field_values[field[:key]] = value
+      end
+      custom_field_values
+    end
+
+    def find_object_by_title(custom_fields, title)
+      custom_fields.find { |f| f[:title_multiloc][@locale] == title }
+    end
+
```

Comment:
This is the last bit of code I wrote to format the custom field values. Probably not so elegant but seems to work (for New Westminster data that we'll be importing), though could do with a thorough check.